jerry-core/ecma/base/ecma-alloc.h

```diff
@@ -30,108 +30,108 @@
  *
  * @return pointer to allocated memory
  */
-extern ecma_object_t *ecma_alloc_object (void);
+extern ecma_object_t * ecma_alloc_object (void);
```

Comment:
@galpeter, I'm not sure if there is determined style requirement for the case.

The main purpose of the pull request is removal of argument names in prototypes of functions that are declared in headers and implemented somewhere in modules, so having duplicate specification of argument names.

For me, all versions of asterisk alignment seem to be acceptable, although I would prefer alignment to left in return values and argument lists. However, I agree with you that style guide with list of all style requirements would be great.

@egavrin, could you, please, clarify your opinion on the case, and your opinion about creating written style guide listing all style requirements?